### ISSUE TITLE
No syntax highlighting for VBScript?

### ISSUE THREAD

**Issue by hillscottc** [HappenApps/Quiver#652]
Alas, I need to use VBScript for an old ASP app.I am not seeing any syntax highlighting at all. :(
Any help?   (The javascript and others langs I've tried works fine.)

<pre>
```vbscript
'Returns a recordset given parameterized sql and an array of params.
Function MyFunc()
    Set MyFunc = "OK"
End Function
```

```VBscript
'Returns a recordset given parameterized sql and an array of params.
Function MyFunc()
    Set MyFunc = "OK"
End Function
```
</pre>

**Comment by hillscottc** [comment]
Argh, and it works in Marxico, which I would prefer not to use.

**Comment by ylian** [comment]
Yeah, it seems that it's supported in the code cell, but left out from the code blocks in Markdown. I will fix.

**Comment by ylian** [comment]
Fixed in 3.0.4.
